src/shared/hooks/use-editor.ts

```diff
@@ -0,0 +1,14 @@
+import { useRef } from "react";
+
+import { MDXEditorMethods } from "@mdxeditor/editor";
+
+export const useEditor = () => {
+  const ref = useRef<MDXEditorMethods>(null);
+  const getResult = () => {
+    if (ref.current) {
+      return ref.current.getMarkdown();
+    }
+    return null;
```

Comment:
```suggestion
    return ref.current ? ref.current.getMarkdown() : "";
```

문자열을 반환한다는 결과값을 유지하기 위한 삼항 리턴은 어떠한가요?

Reply:
ref.current가 유효하지않다는것은 ref가 전달되지 않았다는 것이고
ref가 전달되지 않았다면 useEditor훅이 기대하는 정상적인 동작이 불가해진다는 것이기때문에 
반환은 null 혹은 error를 리턴하는게 맞다고 생각했어요